README.md

```diff
@@ -1,5 +1,51 @@
 # ensembl-client
 
-This repository provides an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://2020.ensembl.org
+## Introduction
 
-If you are interested in the latest developments in this project, please join our Slack channel #ensembl2020
+Ensembl is an open-source, open-access bioinformatics project based at [EMBL-EBI](https://www.ebi.ac.uk) which creates, integrates and distributes genomic data and tools. The main entry point is our website, which provides visualisations of our genomic data as well as online tools for analysing your own data.
+
+This repository is an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://2020.ensembl.org. The new site will integrate genomes from across the taxonomic space, including vertebrates, invertebrates, plants, fungi and bacteria.
+
+## Architecture
+
+The Ensembl client consists of a number of separate apps, each of which relies on one or more backend services to supply content. It is our intention that the Ensembl client code will be reusable in part or whole by other websites.
```

Comment:
> The Ensembl client consists of a number of separate apps

Although Andrea uses the word "apps", this is a misleading word suggesting that the client is put together as a collection of individual microfrontends; while in fact it's just a boring monolith 😄 What is listed below are more likely "pages" (or "sections"? "compartments"?) of the site.

Reply:
I thought the code was intended to be usable as "widgets", e.g. would have the ability to embed the browser in another site. That was the impression I got from the initial launch at whatever conference Andy spoke at.

Reply:
Genome browser (the Rust/wasm part) is indeed very independent and indeed, technically, can be embedded in a different site (although there is some internal knowledge required for making it work). As for the remaining portion of the site, it is not widgety, or embeddable at all, the way it's currently being developed 🤷‍♂ I don't know whether someone should stop us and reassess the project.

Reply:
OK, I'll leave out the apps/embed stuff, and we can raise this at a face-to-face meeting.

Reply:
Reworded.